src/web/assets/garnish/src/MenuBtn.js

```diff
@@ -255,11 +271,33 @@ export default Base.extend(
     },
 
     enable: function () {
-      this.disabled = false;
+      if (!this.$btn) {
+        return;
+      }
+
+      this.$btn.removeAttr('disabled');
     },
 
     disable: function () {
-      this.disabled = true;
+      if (!this.$btn) {
+        return;
+      }
+      console.log('disable this');
+      this.$btn.attr('disabled', 'disabled');
+    },
+
+    handleStatusChange: function () {
+      if (!this.$btn) {
+        return;
+      }
+
+      if (this.$btn.attr('disabled') === 'disabled') {
```

Comment:
Should probably go with `Garnish.hasAttr()` here rather than worrying about the specific value.

Reply:
I didn't know that method existed in Garnish. Have updated.